.github/workflows/test.yml

```diff
@@ -90,7 +90,7 @@ jobs:
           $CHISEL_FIRTOOL_PATH/firtool -version >> $GITHUB_STEP_SUMMARY
           echo \`\`\` >> $GITHUB_STEP_SUMMARY
       - name: Test
-        run: ./mill -j0 firrtl.cross[].test + svsim.cross[].test + chisel[].test
+        run: ./mill -j0 firrtl.cross[].test + svsim.cross[].test + chisel[].test -oF
```

Comment:
This is probably only gonna affect the chisel tests and not firrtl nor svsim. Probably fine but just FYI

Reply:
I didn't understand this syntax. Thanks! Changed in https://github.com/chipsalliance/chisel/pull/4707/commits/9cfb32b082f65c75ceb7538401a447f18e2d4354.